README.md

```diff
@@ -35,6 +45,28 @@
 - [ ] Release a production ready service
 - [ ] Release a beta of the style analyzer[s]
 
+### Code review study across GitHub
+
+- [ ] Conduct the study how people do code reviews on GitHub
+    - We need the numbers. E.g. what is the ratio of stupid style fixes compared to the overall? Clever fixes?
+
+### ML approach to source code style check
+
+- [ ] General code style rules mining under AST
+- [ ] General code style rules mining over AST
+    - What does this mean? Read https://github.com/src-d/minutes/blob/673545e3c9d2bc12a2f112f725cccec6d8054ff4/machine-learning/2018-06-22-Lookout-discussion.md
+- [ ] Naming style suggestion
+- [ ] Bad names detection
+- [ ] Typos correction in names
+- [ ] One-line best practices mining
+
+### ML support objectives
+
+- [ ] Maintain the sourced-ml stack
+- [ ] Incorporate the identifier splitter inside sourced-ml extraction pipeline
+    - Yes, Máximo, it is really needed for the upstream tasks suchs as typos, naming style and bad names detection
+- [ ] Internal ML research meetups once per month
```

Comment:
This is a internal OKR for your team, not engineering wise.

Reply:
Just tell me where to put it under.

Reply:
How this contributes to company OKRs?

Reply:
Without these meetups there is much less knowledge exchange which leads to not efficient work of ML team. All the leading ML teams which I know from Google, Yandex, Mail.ru, Samsung have these. It is the only way to discuss the science and how it can be applied to a particular project. In our case - intelligent code review. It also includes overviews of the new bleeding edge research.

Reply:
This is an internal procedure of your team, I don't think that belongs to this document, in the same way, that the Apps Dinners are not written here. Or that we are having a hackathon in September.